CHANGELOG.md

```diff
@@ -1,4 +1,146 @@
 # Changelog
+ 
+## [v0.1.0](https://github.com/lf-lang/lingua-franca/tree/v0.1.0) (2022-04-04)
```

Comment:
The changelog refers to version 0.1.0

Reply:
Right. This is as it should be. The changelog only includes changes that have been released.